diff --git a/node_modules/nofilter/lib/index.js b/node_modules/nofilter/lib/index.js
index 89bceb5..a2be440 100644
--- a/node_modules/nofilter/lib/index.js
+++ b/node_modules/nofilter/lib/index.js
@@ -2,7 +2,7 @@
 
 const stream = require('stream')
 const {Buffer} = require('buffer')
-const td = new TextDecoder('utf8', {fatal: true, ignoreBOM: true})
+const td = new TextDecoder('utf8', {fatal: false, ignoreBOM: true})
 
 /**
  * @typedef {object} NoFilterOptions
